vignettes/ExtendingGenomicRanges.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "**Extending Genomic Ranges**"
```

Comment:
Please change the title to: 

title: "Extending *GenomicRanges*"

Reply:
Here is a screenshot of the formatting: 
<img width="262" alt="Screen Shot 2022-11-05 at 12 19 19 AM" src="https://user-images.githubusercontent.com/105299892/200100439-836d9445-d33b-4a8e-9fea-9b5887c9f81d.png">